src/types/systemIntake.ts

```diff
@@ -5,12 +5,43 @@ export type GovernanceCollaborationTeam = {
   name: string;
 };
 
-export type SystemIntakeStatus =
-  | 'DRAFT'
-  | 'SUBMITTED'
-  | 'ACCEPTED'
-  | 'APPROVED'
-  | 'CLOSED';
+export const openIntakeStatuses = [
```

Comment:
Below are arrays of possible intake statuses:
1. Open Statuses
2. Closed Statuses
3. Old Statuses

The old statuses ensure the application wont break for now, but they should be deleted once deprecated.